<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colossal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="s_mainWindowGridLayout">
    <item row="1" column="1">
     <widget class="QGroupBox" name="s_transformGB">
      <property name="title">
       <string>Transform</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Draw</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QComboBox" name="m_objectSelection">
         <item>
          <property name="text">
           <string>Teapot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sphere</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cube</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="m_wireframe">
         <property name="text">
          <string>Wireframe</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>43</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
